test/unit/util/errors/PayloadHttpError.test.ts

```diff
@@ -0,0 +1,27 @@
+import { BadRequestHttpError } from '../../../../src/util/errors/BadRequestHttpError';
+import { PayloadHttpError } from '../../../../src/util/errors/PayloadHttpError';
+
+describe('PayloadHttpError', (): void => {
```

Comment:
This file isn't actually needed, you can just add it to the list in the HttpError [test](https://github.com/solid/community-server/blob/1a50df25ec7e8cbc5a099ba0c81433b6fe54587f/test/unit/util/errors/HttpError.test.ts#L22)